saleor/static/scss/components/_header.scss

```diff
@@ -209,7 +209,7 @@
         vertical-align: middle;
       }
       .btn {
-        padding: $global-padding;
+        padding: $global-padding / 2;
```

Comment:
This indeed fixes the search button, but after the Bootstrap upgrade, the whole search bar is narrower than it used to be. This is how it looked (my devtools shows that the height was 53px):

![image](https://user-images.githubusercontent.com/5421321/53174218-941b9b00-35e9-11e9-91ac-60f3b98df5ab.png)

Can we revert the height of that input?

Reply:
Will do 👍